cluster/manager/manager.go

```diff
@@ -427,8 +427,12 @@ func (c *ClusterManager) getNonDecommisionedPeers(
 		if nodeEntry.Status == api.Status_STATUS_DECOMMISSION {
 			continue
 		}
+		dataIp := nodeEntry.DataIp
```

Comment:
suggest writing a function and using it instead of repeating code:

```
func getPeerAddress(ip string, port string) string
{
      if strings.Contains(ip, ":") {
 			ip = fmt.Sprintf("[%s]", ip)
      }
      return fmt.Sprintf("%s:%s", ip, port)
}
```